extensions-core/azure-extensions/pom.xml

```diff
@@ -159,7 +159,7 @@
             <plugin>
                 <groupId>org.jacoco</groupId>
                 <artifactId>jacoco-maven-plugin</artifactId>
-                <version>0.8.4</version>
+                <version>${jacoco.version}</version>
```

Comment:
IIRC, if the `<version>` is in `<pluginManagement>` in the main pom, then the line doesn't need to be present at all here.

Reply:
Removed.